lib/reversetunnel/seek/seek.go

```diff
@@ -248,6 +250,10 @@ func (p *proxyGroup) run(ctx context.Context) {
 			if stat.ShouldSeek() {
 				p.notifyShouldSeek()
 			}
+			ticks++
+			if ticks%8 == 0 {
```

Comment:
What is `ticks%8`?

Reply:
`gofmt` keeps removing the space around the remainder operator.  The above code is limiting log spam by only logging state once every 8 ticks.